libdevcore/CommonIO.cpp

```diff
@@ -26,6 +26,43 @@
 using namespace std;
 using namespace dev;
 
+std::list<std::list<std::string>> dev::memDumpToList(bytes const& _b, unsigned _w)
```

Comment:
Oh, didn't realise that. In that case I'd suggest moving it to mix codebase as it is really too specific